src/rules/consistent-type-specifier-style.js

```diff
@@ -0,0 +1,227 @@
+import docsUrl from '../docsUrl';
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Enforce or ban the use of inline type-only markers for named imports',
+      url: docsUrl('consistent-type-specifier-style'),
+    },
+    fixable: 'code',
+    schema: [
+      {
+        type: 'string',
+        enum: ['prefer-inline', 'prefer-top-level'],
+        default: 'prefer-inline',
+      },
+    ],
+  },
+
+  create(context) {
+    const sourceCode = context.getSourceCode();
+
+    if (context.options[0] === 'prefer-inline') {
+      return {
+        ImportDeclaration(node) {
+          if (node.importKind === 'value' || node.importKind == null) {
+            return;
+          }
+
+          if (
+            // no specifiers (import type {} from '') have no specifiers to mark as inline
+            node.specifiers.length === 0 ||
+            (node.specifiers.length === 1 &&
+              // default imports are both "inline" and "top-level"
+              (node.specifiers[0].type === 'ImportDefaultSpecifier' ||
+                // namespace imports are both "inline" and "top-level"
+                node.specifiers[0].type === 'ImportNamespaceSpecifier'))
+          ) {
+            return;
```

Comment:
can this line be covered by a test case?

Reply:
This should already be covered by:
- https://github.com/bradzacher/eslint-plugin-import/blob/2df9be6b7edd150c9ac94b2d479ea51f5fbe66d0/tests/src/rules/consistent-type-specifier-style.js#L16-L19
- https://github.com/bradzacher/eslint-plugin-import/blob/2df9be6b7edd150c9ac94b2d479ea51f5fbe66d0/tests/src/rules/consistent-type-specifier-style.js#L40-L43
- https://github.com/bradzacher/eslint-plugin-import/blob/2df9be6b7edd150c9ac94b2d479ea51f5fbe66d0/tests/src/rules/consistent-type-specifier-style.js#L207

and a few others

Reply:
if it were, then i wouldn't expect to see the "no coverage" annotation

Reply:
Weird - I'm not sure what's happening with the coverage report.
Locally I can see it's getting hit in the tests:
<img width="1117" alt="image" src="https://user-images.githubusercontent.com/7462525/188565754-a39b9f0a-c4c5-4a45-8f1e-5ca8fc965866.png">